Makefile

```diff
@@ -5,22 +5,27 @@ SPARQL=sparql
 ROBOT_ENV=ROBOT_JAVA_ARGS=-Xmx12G
 ROBOT=$(ROBOT_ENV) robot
 
-all: $(BUILD_DIR)/phenoscape-ontology-classified.ofn
+all: $(BUILD_DIR)/phenoscape-kb.ttl $(BUILD_DIR)/phenoscape-kb-tbox-hierarchy.ofn $(BUILD_DIR)/qualities.txt $(BUILD_DIR)/anatomical_entities.txt
 
 clean:
 	rm -rf $(BUILD_DIR)
 
+# Create build directory
+$(BUILD_DIR): clean
```

Comment:
I don't think this should depend on `clean`. Any reason why?

Reply:
I was just trying to make deleting the directory a prerequisite of creating the directory. That was we are always sure it is a clean start.